constellations/orion/src/domains/class.go

```diff
@@ -23,6 +23,7 @@ type Class struct {
 	StartDate       time.Time  `json:"startDate" db:"start_date"`
 	EndDate         time.Time  `json:"endDate" db:"end_date"`
 	GoogleClassCode NullString `json:"googleClassCode" db:"google_class_code"`
+	PriceLump       uint       `json:"priceLump" db:"price_lump"`
```

Comment:
I'm not sure if I mentioned this, apologies if I didn't. PriceLump is optional and can be NULL. So PriceLump type should be NullUint.